### PR TITLE
Localize toasts on hooks

### LINKED ISSUE #77: Localize Toast Message
Translation support for the toast section that displays the results of the response from the contract.
Note that it does not work well as usual because the information cannot be successfully transmitted from the page to the hooks. Investigation required.

### PR DESCRIPTION
**Pull request by alecrem**
Closes #77

- Localized toasts from keyword hook
- Localized toasts from claim hook

### CODE REVIEW

File: locales/en/common.json [hunk 1]
```diff
@@ -52,6 +52,17 @@
...
+    "toast": {
+      "submitted": "Submitted",
+      "error": "Error",
+      "success": "You got $HENKAKU.",
+      "insufficientAmount": "You don't have points to claim $HENKAKU.",
+      "insufficientFound": "Currently unable to claim.",
+      "txFailed": "Transaction failed.",
+      "defaultError": "Unknown Server Error"
+    }
+  },
```
Comment (alecrem) [inline locales/en/common.json:65]: To be moved to the claim namespace after https://github.com/henkaku-center/omise-interface/pull/114 is merged

File: locales/en/common.json [hunk 2]
```diff
@@ -66,6 +77,15 @@
...
     "TOAST": {
+      "keyword": {
+        "submitted": "Submitted",
+        "error": "Error",
+        "correct": "Your answer was correct.\nYou earn 100 points.",
+        "mustBeHolder": "Only Kamon holders may answer quests.",
+        "wrongAnswer": "Your answer is wrong.",
+        "alreadyAnswered": "You have already answered this week's question.",
+        "defaultError": "Unknown Server Error"
+      },
```
Comment (alecrem) [inline locales/en/common.json:88]: All quest strings will be moved to their own namespace after #116 is merged.

File: locales/ja/common.json [hunk 3]
```diff
@@ -66,6 +77,15 @@
     "INPUT_PLACEHOLDER": "キーワード",
     "SUBMIT_BUTTON": "キーワードを送信",
     "TOAST": {
+      "keyword": {
+        "submitted": "Submitted",
+        "error": "Error",
+        "correct": "回答は正解でした。\n100ポイントを獲得しました。",
```
Comment (hourglasshoro) [inline locales/ja/common.json:83]: [IMO] Maybe the line breaks may not work.

**Review by hourglasshoro — COMMENTED** [review]
I have checked the operation and confirmed that it is translated

**Review by hourglasshoro — APPROVED** [review]
LGTM!

**Review by geeknees — APPROVED** [review]

**Pull request merged**